test/integration/production/test/index.test.js

```diff
@@ -417,7 +417,9 @@ describe('Production Usage', () => {
     if (browserName === 'chrome') {
       it('should add preload tags when Link prefetch prop is used', async () => {
         const browser = await webdriver(appPort, '/prefetch')
+        await waitFor(50)
```

Comment:
Let's just make this 2000 and call it a day 😛 
```suggestion
        await waitFor(200)
```